neon_core/run_neon.py

```diff
@@ -136,7 +136,7 @@ def _stop_all_core_processes():
                 psutil.Process(pid).terminate()
                 sleep(1)
                 if psutil.pid_exists(pid) and psutil.Process(pid).is_running():
-                    LOG.error(f"Process {pid} not terminated!!")
+                    LOG.info(f"Process {pid} not terminated!!")
```

Comment:
shouldnt this stay as error level?

Reply:
The process is killed here and the skills module always logs this.. Maybe a warning?